README.md

```diff
@@ -28,5 +32,76 @@ module "codebuild_webdeploy_app_name" {
   build_configuration    = "Release"
   input_tags             = local.common_tags
 }
-
 ```
+## StratusGrid Standards we assume
+- All resource names and name tags shall use `_` and not `-`s
+- The old naming standard for common files such as inputs, outputs, providers, etc was to prefix them with a `-`, this is no longer true as it's not POSIX compliant. Our pre-commit hooks will fail with this old standard.
+- StratusGrid generally follows the TerraForm standards outlined [here](https://www.terraform-best-practices.com/naming)
+## Repo Knowledge
+Repository for Module vmimport
```

Comment:
I don't think this is the correct module name. See comment on the config file.

Reply:
Fixed